### DISCUSSION TITLE
Locators to quickly snap blocks in place

### DISCUSSION THREAD

**Discussion by benngarnish** [jjburton/cgmTools#291] [Ideas]
This is something towards the configuration process and ease of building setups that can be shared.
Have a system of locators that you can place around the character(biped in this case) and then load/import blocks that you have setup previously. Then by running a simple script these blocks snap into place based on the locators location.
This will speed up workflow and also allow for iteration.

**Comment by jjburton** [comment]
this is neat, Benn. Keep your notes here so others can follow along.
